jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/ButtonPage.java

```diff
@@ -3,37 +3,34 @@
 import com.epam.jdi.light.elements.composite.WebPage;
 import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
 import com.epam.jdi.light.material.annotations.*;
-import com.epam.jdi.light.material.elements.inputs.Button;
+import com.epam.jdi.light.ui.html.elements.common.Button;
```

Comment:
Do you really don't need material Button import?

Reply:
I used only html Button import, and don't need to use Button inport form material